catroid/src/main/java/org/catrobat/catroid/ui/BaseActivity.java

```diff
@@ -174,6 +178,15 @@ private boolean isRecoveredFromCrash() {
 		return PreferenceManager.getDefaultSharedPreferences(this).getBoolean(RECOVERED_FROM_CRASH, false);
 	}
 
+	private Intent createSettingsIntent() {
```

Comment:
Yes, that's one thing I've planned to refactor, too.
OK, then I'll refactor it directly in this branch.